src/Servers/Kestrel/Kestrel/src/WebHostBuilderKestrelExtensions.cs

```diff
@@ -46,7 +46,10 @@ public static IWebHostBuilder UseKestrel(this IWebHostBuilder hostBuilder)
             options.DefaultCloseErrorCode = (long)Http3ErrorCode.NoError;
         });
 
-        hostBuilder.UseNamedPipes();
+        if (OperatingSystem.IsWindows())
+        {
+            hostBuilder.UseNamedPipes();
```

Comment:
this applies to the original, but for my education, does this line mean named pipes are simply enabled as an option, not that they are somehow the default transport?
(if the latter, perhaps EnableNamedPipes() would be a better name, but perhaps UseXX() is the established pattern)

Reply:
UseXXX is the established pattern. AddXXX would be better but the current pattern dates from when there was one transport at a time. Stuck with UseXXX to be consistent.